tests/unit_tests/regression/test_aerodyn_inflow.cpp

```diff
@@ -0,0 +1,40 @@
+#include <gtest/gtest.h>
+
+#include "test_utilities.hpp"
+
+#include "src/vendor/dylib/dylib.hpp"
+
+namespace openturbine::tests {
+
+TEST(AerodynInflowTest, ADI_C_PreInit) {
```

Comment:
For this test to run in CI, we'll need to make sure that the `libaerodyn_inflow_c_binding.*` file is in the same directory as the unit test executable.

Reply:
Also, this test should be disabled if we aren't building with aerodyn support

Reply:
I added conditional support for the test, will wait to see the CI outputs before making any changes to the workflow there.

Reply:
Sounds good